staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -362,6 +375,14 @@ func builtinTagDocs() []validators.TagDoc {
 			Description: "<field-name>",
 			Docs:        "This values names a field of a list's value type.",
 		}},
+	}, {
+		Tag:         subfieldTag,
```

Comment:
Since this is the first such, perhaps we should document it as `fmt.Sprintf("%s(<field-name>)")` ?

Reply:
Ah good point, I've updated this to now be:
```
		Tag:         fmt.Sprintf("%s(<field-name>)", subfieldTag),
```